source/features/edit-readme.tsx

```diff
@@ -10,13 +10,10 @@ import {isPermalink} from '../github-helpers';
 import getDefaultBranch from '../github-helpers/get-default-branch';
 
 async function init(): Promise<void | false> {
-	const readmeHeader = await elementReady('#readme .Box-title');
-	if (!readmeHeader) {
-		return false;
-	}
+	const readmeHeader = (await elementReady('#readme :is(.Box-header, .js-sticky)'))!;
 
 	const isPermalink_ = await isPermalink();
-	const filename = readmeHeader.textContent!.trim();
+	const filename = select('[href="#readme"]')!.textContent!.trim();
```

Comment:
This 👍 I also thought about appending to header, but was hesitant bacause of the usage of `readmeHeader` here.